csharp/test/Ice/metrics/AllTests.cs

```diff
@@ -356,7 +357,7 @@ private static void
         }
     }
 
-    private static async void connect(Ice.ObjectPrx proxy)
```

Comment:
This was the root issue: connect is now async but we no longer waited for connect to complete.